### DISCUSSION TITLE
New Money wallet rejected (Project Submission)

### DISCUSSION THREAD

**Discussion by ArtemSirobaba** [WalletConnect/walletconnect-monorepo#3631] [Developer Support] [answered]
Hello, I am developing New Money wallet (Project ID: 34939ea651efba8e412d10c6e1928b6b)
All functionalities are working as expected here https://lab.web3modal.com/.
I have submitted the wallet for review but it has been rejected a few times.
Could you please re-evaluate my request or provide feedback on what may be the issue?

Reason:
"Wallet Testing - Could not verify if testing has been done on this wallet. As we do not have available V2 metric data for Sign. Please test your integration with our test dapp and submit for approval if successful. Link: https://lab.web3modal.com/"

**Comment by glitch-txs** [comment]
Hi, have you been able to connect to labs with your wallet and sign?

**Reply by ArtemSirobaba** [reply]
Yes, I have been able to connect and test wallet (Sign v2 only works for mobile wallet)

**Reply by JonathanConn** [reply]
Could you test the wallet with https://lab.web3modal.com/ using your mobile wallet for us?
We need to be able to verify a v2 sign request to publish your submission

**Reply by ArtemSirobaba** [reply, marked as answer]
Done, tested it as well) Re-opened submission again

**Reply by JeroldG** [reply]
Thanks ! 

Your request is now approved, please allow 60 minutes for changes to propagate. 

Best,
jerold

**Reply by ArtemSirobaba** [reply]
Thank you!
